include/swift/AST/ProtocolConformanceRef.h

```diff
@@ -111,8 +111,9 @@ class ProtocolConformanceRef {
 
   /// Replace opaque types in the conforming type with their underlying types,
   /// and resolve opaque conformances to their underlying conformances.
-  ProtocolConformanceRef substOpaqueTypesWithUnderlyingTypes(Type origType) const;
-  
+  ProtocolConformanceRef substOpaqueTypesWithUnderlyingTypes(
+      Type origType, ModuleDecl *modulePerformingSubstitution) const;
```

Comment:
Can you move this method, SubstitutionMap::substOpaqueTypesWithUnderlyingTypes() and ReplaceOpaqueTypesWithUnderlyingTypes to your SIL pass since that's the only place where they're used now?